requirements-minimal.txt

```diff
@@ -49,7 +49,7 @@ requests-cache >= 0.4.10
 retry
 ruamel.yaml
 sensu-plugin
-service-configuration-lib >= 2.16.4
+service-configuration-lib >= 2.17.0
```

Comment:
this may change if someone else makes a release without https://github.com/Yelp/service_configuration_lib/pull/115